src/parser/scanner_support.c

```diff
@@ -676,6 +692,17 @@ pt_check_hint (const char *text, PT_HINT hint_table[], PT_HINT_ENUM * result_hin
 			      if (arg)
 				{
 				  temp = strstr (arg_start, ".");
+
+				  /*
+				   * "user_name + dot(.)" is added in front of the table name to distinguish
+				   * the user schema. So we cannot separate index names with dot(.).
+				   * dot(.) does not appear more than 2 times, and the last name is the index name.
+				   */
+				  if (temp && strstr (temp + 1, ".") != NULL)
+				    {
+				      temp = strstr (temp + 1, ".");
+				    }
```

Comment:
`strstr()` is performed twice unnecessary.

Reply:
Looking back now, I don't need to add this code.
I will comment again after more checking.

Reply:
I changed it to enable the query below.
```
select /*+ select_key_info (dba.t1.pk_t1_c1) */ * from t1;
```
However, I think it would be better to just write the index name, so I delete the related changes.